spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/batch/AbstractSimpleJobServiceTests.java

```diff
@@ -86,7 +86,7 @@ public abstract class AbstractSimpleJobServiceTests extends AbstractDaoTests {
 	protected void prepareForTest(JdbcDatabaseContainer dbContainer, String schemaName, DatabaseType databaseType) throws Exception {
 		this.databaseType = databaseType;
 		super.prepareForTest(dbContainer, schemaName);
-		this.jdbcSearchableJobExecutionDao = new JdbcSearchableJobExecutionDao();
+		this.jdbcSearchableJobExecutionDao = new JdbcSearchableJobExecutionDao(BatchVersion.BATCH_4);
```

Comment:
Can we add tests for the BATCH_5 as well?

Reply:
Only with a lot of manual SQL statements.

Reply:
Turns out jobs are already created for boot2 and boot3 and the queries uses the SimpleJobService where the modified DOAs are used.